src/transformers/integrations/integration_utils.py

```diff
@@ -1002,7 +1009,7 @@ def setup(self, args, state, model):
         # "synchronous" flag is only available with mlflow version >= 2.8.0
         # https://github.com/mlflow/mlflow/pull/9705
         # https://github.com/mlflow/mlflow/releases/tag/v2.8.0
-        if packaging.version.parse(importlib.metadata.version("mlflow")) >= packaging.version.parse("2.8.0"):
+        if get_mlflow_version() >= packaging.version.parse("2.8.0"):
```

Comment:
Here will need the "packaging.version.parse" to make the returned `str` as a Version so that they can be compared

Reply:
Done, @cchen-dialpad